unabridged-works-volume-3/original/13-sensible-weighty-queries.adoc

```diff
@@ -269,57 +302,76 @@ The Lord God of everlasting power break down the kingdom of Satan
 and exalt the kingdom and sceptre of his own Son instead thereof.
 Amen.
 
-=== POSTSCRIPT / CONTAINING SOME QUERIES ON Isa. 50:10-11
+[.old-style]
```

Comment:
Wasn't sure if this was right for this postscript.

Reply:
i think you did it right 👍